app/Services/Service.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Services;
+
+use App\Core\Db;
+
+/**
+ * Base Service
+ */
+abstract class Service {
+    
+    protected $Db;
```

Comment:
not critical but usually class property starts from lowercase, please rename $Db => $db